.github/ISSUE_TEMPLATE/documentation_request.yml

```
@@ -0,0 +1,64 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Documentation Request
+description: Request updates or additions to cuDF's documentation
+title: "[DOC]: "
+labels: ["doc"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to help cuDF and improve our documentation!
+        - By submitting this issue, you agree to follow our [Code of Conduct](https://docs.rapids.ai/resources/conduct/)
+        - You also confirm that you have searched the [open documentation issues](https://github.com/rapidsai/cudf/issues?q=is%3Aopen+is%3Aissue+label%3Adoc) and have found no duplicates for this request
+
+  - type: dropdown
+    id: criticality
+    attributes:
+      label: How would you describe the priority of this documentation request?
+      options:
+        - Critical (currently preventing usage)
+        - Medium
+        - Low (would be nice)
+    validations:
+      required: true
+
+  - type: dropdown
+    id: new_or_correction
+    attributes:
+      label: Is this for new documentation, or an update to existing docs?
+      options:
+        - New
+        - Update
+    validations:
+      required: true
+
+  - type: textarea
+    id: problem
+    attributes:
+      label: Describe the incorrect/future/missing documentation
```

```suggestion
      label: Describe the incorrect/missing documentation
```

Rationale: I have no idea what "future" documentation is, if not "missing".

---

Intention is that:
- Incorrect: wrong/out of date documentation
- Missing: there's functionality but no docs
- Future: what's there isn't wrong or "missing" but the docs don't capture what I want
   -  Things like improved readmes, user guides, contrib guides would fall here

That being said, totally fine removing.

---

Given that, I would class future as a subset of missing.